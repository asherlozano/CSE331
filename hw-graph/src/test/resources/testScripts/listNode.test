# listNode test file
# tests if you the listNodes method can be added to graph
# listNodes lists all the nodes of a graph

# Create a graph
CreateGraph  g1

# Create a node
AddNode g1 n1

# Print the graphs nodes
ListNodes g1