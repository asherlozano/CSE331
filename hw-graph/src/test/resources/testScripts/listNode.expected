# listNode expected file
# tests if you can listNodes method can be added to graph
# listNodes lists all the nodes of a graph

# Create a graph
created a graph g1

# Create a node
added node n1 to g1

# Print the nodes in the graph
g1 contains: n1